aws/sdk/integration-tests/s3/tests/endpoints.rs

```diff
@@ -3,6 +3,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#![cfg(feature = "test-util")]
```

Comment:
Now that integration tests are predicated on features, we should update the `check-aws-sdk-standalone-integration-tests` script here:
```
pushd "${tmp_dir}/aws/sdk/integration-tests"
cargo check --tests
popd
```
It should check with `--all-features`.